packages/app/src/server/routes/apiv3/forgot-password.js

```diff
@@ -81,13 +81,20 @@ module.exports = (crowi) => {
     }
   });
 
+  // TODO: grune - make new route get for `/forgot-password/activate/${passwordResetOrderData.token}`
+  // Adjust based on this route router.put('/', ...);
+
   router.put('/', injectResetOrderByTokenMiddleware, async(req, res) => {
+
+    console.log('grunelog URL token action');
+
```

Comment:
Not good to code in this file because user registration is different from the forgot-password method.
please create a new file under the apiv3 directory

Also, please do not include console.log in a PR...!
Thank you.

Reply:
Done on this commit: 224c6f051f1dc4df811e6b5aad776f9562cc5335